libpod/boltdb_state.go

```diff
@@ -460,7 +461,7 @@ func (s *BoltState) UpdateContainer(ctr *Container) error {
 			// Open the new network namespace
 			ns, err := joinNetNS(netNSPath)
 			if err != nil {
-				return errors.Wrapf(err, "error joining network namespace for container %s", ctr.ID())
+				logrus.Errorf("error joining network namespace for container %s", ctr.ID())
 			}
 			newState.NetNS = ns
```

Comment:
Only do this if we didn't error, so trying to remove the network namespace won't blow up as well.